nixos/hosts/moss/default.nix

```diff
@@ -6,6 +6,7 @@
   imports = [
     ../../configuration.nix
     ../../gui.nix
+    ../../hardware.nix
```

Comment:
I should think how to realize `moss` specific keyremapper and merging it with this with 99-local.hwdb

Reply:
```
130! udevadm info --attribute-walk /dev/input/event1 | grep -F 'ATTRS{name}'
    ATTRS{name}=="AT Translated Set 2 keyboard"

> nix run nixpkgs#evtest -- /dev/input/event1
Testing ... (interrupt to exit)
Event: time 1726425659.538497, type 4 (EV_MSC), code 4 (MSC_SCAN), value 3a
Event: time 1726425659.538497, type 1 (EV_KEY), code 58 (KEY_CAPSLOCK), value 1
Event: time 1726425659.538497, -------------- SYN_REPORT ------------
Event: time 1726425659.538715, type 17 (EV_LED), code 1 (LED_CAPSL), value 1
```

Reply:
https://github.com/kachick/dotfiles/commit/5bd7927e2e4bef6ea83608f47ffc355aa89a62e7